<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QMainWindow" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListView" name="listView"/>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>No Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Playback Mode:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="playbackMode">
      <item>
       <property name="text">
        <string>CurrentItemOnce</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CurrentItemInLoop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Linear</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Loop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Random</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Add images</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Shuffle</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>188</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonBack">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdvance">
        <property name="text">
         <string>Advance</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadPlaylist"/>
    <addaction name="actionSavePlaylist"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadPlaylist">
   <property name="text">
    <string>Load playlist...</string>
   </property>
  </action>
  <action name="actionSavePlaylist">
   <property name="text">
    <string>Save playlist...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ImageViewer</receiver>
   <slot>addImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBack</sender>
   <signal>clicked()</signal>
   <receiver>ImageViewer</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdvance</sender>
   <signal>clicked()</signal>
   <receiver>ImageViewer</receiver>
   <slot>advance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ImageViewer</receiver>
   <slot>shuffle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoadPlaylist</sender>
   <signal>triggered()</signal>
   <receiver>ImageViewer</receiver>
   <slot>loadPlaylist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSavePlaylist</sender>
   <signal>triggered()</signal>
   <receiver>ImageViewer</receiver>
   <slot>savePlaylist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>ImageViewer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playbackMode</sender>
   <signal>activated(int)</signal>
   <receiver>ImageViewer</receiver>
   <slot>setPlaybackMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addImage()</slot>
  <slot>advance()</slot>
  <slot>back()</slot>
  <slot>shuffle()</slot>
  <slot>loadPlaylist()</slot>
  <slot>savePlaylist()</slot>
  <slot>setPlaybackMode(int)</slot>
 </slots>
</ui>
